### PR TITLE
ui: Remove monorepo

### PR DESCRIPTION
**Pull request by breeswish**
This PR removes the monorepo to make life easier.

Fix https://github.com/pingcap-incubator/tidb-dashboard/issues/324
Fix https://github.com/pingcap-incubator/tidb-dashboard/issues/280
Fix https://github.com/pingcap-incubator/tidb-dashboard/issues/269
Fix https://github.com/pingcap-incubator/tidb-dashboard/issues/170
Fix https://github.com/pingcap-incubator/tidb-dashboard/issues/75

Also fix:

- Some styles and scripts were imported / executed multiple times
- Some package stylesheets did not have Dashboard global theme color
- Could not use `@import '~antd/es/...` in packages
- Could not import `@umijs/hooks` in packages
- KeyViz produced several lint warnings

### CODE REVIEW

File: ui/dashboardApp/registry.js [hunk 1]
```diff
@@ -9,6 +12,26 @@ export default class AppRegistry {
...
+        return component
+      },
+      domElementGetter: () => document.getElementById(targetDomId),
+    })
+    return {
+      bootstrap: [reactLifecycles.bootstrap],
+      mount: [reactLifecycles.mount],
+      unmount: [reactLifecycles.unmount],
+    }
+  }
```
Comment (baurine) [inline ui/dashboardApp/registry.js:33]: perfect!

File: ui/lib/apps/Diagnose/components/DiagnoseStatus.tsx [hunk 2]
```diff
@@ -2,11 +2,11 @@ import React, { useState, useEffect } from 'react'
 import { useParams, Link } from 'react-router-dom'
 import { ArrowLeftOutlined } from '@ant-design/icons'
 import { Descriptions, message, Skeleton, Progress, Button } from 'antd'
-import { Head } from '@pingcap-incubator/dashboard_components'
-import { DateTime } from '@/components'
-import { DiagnoseReport } from '@pingcap-incubator/dashboard_client'
+import { Head } from '@lib/components'
+import { DateTime } from '@lib/components'
```
Comment (baurine) [inline ui/lib/apps/Diagnose/components/DiagnoseStatus.tsx:6]: not a big deal, but can be combined, same as the following line 7 and 9

File: ui/lib/apps/InstanceProfiling/pages/Detail.js [hunk 3]
```diff
@@ -4,10 +4,10 @@ import { ArrowLeftOutlined } from '@ant-design/icons'
 import { Button, Badge, Progress } from 'antd'
 import { useTranslation } from 'react-i18next'
 import { Link } from 'react-router-dom'
-import { Head } from '@pingcap-incubator/dashboard_components'
-import { CardTableV2 } from '@pingcap-incubator/dashboard_components'
-import client from '@pingcap-incubator/dashboard_client'
-import { useClientRequestWithPolling } from '@/utils/useClientRequest'
+import { Head } from '@lib/components'
+import { CardTableV2 } from '@lib/components'
```
Comment (baurine) [inline ui/lib/apps/InstanceProfiling/pages/Detail.js:8]: mark

File: ui/lib/apps/InstanceProfiling/pages/List.js [hunk 4]
```diff
@@ -3,10 +3,10 @@ import { message, Form, TreeSelect, Button, Select, Badge } from 'antd'
...
 import { useNavigate } from 'react-router-dom'
 import { ScrollablePane } from 'office-ui-fabric-react/lib/ScrollablePane'
-import { Card, CardTableV2 } from '@pingcap-incubator/dashboard_components'
-import client from '@pingcap-incubator/dashboard_client'
-import { useClientRequest } from '@/utils/useClientRequest'
-import DateTime from '@/components/DateTime'
+import { Card, CardTableV2 } from '@lib/components'
+import client from '@lib/client'
+import { useClientRequest } from '@lib/utils/useClientRequest'
+import DateTime from '@lib/components/DateTime'
```
Comment (baurine) [inline ui/lib/apps/InstanceProfiling/pages/List.js:9]: mark

File: ui/lib/apps/SearchLogs/components/SearchHeader.tsx [hunk 5]
```diff
@@ -1,8 +1,8 @@
-import client from '@pingcap-incubator/dashboard_client'
+import client from '@lib/client'
 import {
   LogsearchCreateTaskGroupRequest,
   LogsearchSearchTarget,
-} from '@pingcap-incubator/dashboard_client'
+} from '@lib/client'
```
Comment (baurine) [inline ui/lib/apps/SearchLogs/components/SearchHeader.tsx:5]: mark

File: ui/lib/apps/SearchLogs/components/SearchHistory.tsx [hunk 6]
```diff
@@ -1,9 +1,6 @@
-import client from '@pingcap-incubator/dashboard_client'
-import {
-  LogsearchSearchTarget,
-  LogsearchTaskGroupResponse,
-} from '@pingcap-incubator/dashboard_client'
-import { CardTable, Head } from '@pingcap-incubator/dashboard_components'
+import client from '@lib/client'
+import { LogsearchSearchTarget, LogsearchTaskGroupResponse } from '@lib/client'
```
Comment (baurine) [inline ui/lib/apps/SearchLogs/components/SearchHistory.tsx:2]: mark

File: ui/lib/apps/SearchLogs/components/SearchProgress.tsx [hunk 7]
```diff
@@ -1,5 +1,5 @@
-import client from '@pingcap-incubator/dashboard_client'
-import { LogsearchTaskModel } from '@pingcap-incubator/dashboard_client'
+import client from '@lib/client'
+import { LogsearchTaskModel } from '@lib/client'
```
Comment (baurine) [inline ui/lib/apps/SearchLogs/components/SearchProgress.tsx:2]: mark

File: ui/lib/apps/SearchLogs/components/SearchResult.tsx [hunk 8]
```diff
@@ -1,9 +1,6 @@
-import client from '@pingcap-incubator/dashboard_client'
-import {
-  LogsearchSearchTarget,
-  LogsearchTaskModel,
-} from '@pingcap-incubator/dashboard_client'
-import { Card } from '@pingcap-incubator/dashboard_components'
+import client from '@lib/client'
+import { LogsearchSearchTarget, LogsearchTaskModel } from '@lib/client'
```
Comment (baurine) [inline ui/lib/apps/SearchLogs/components/SearchResult.tsx:2]: mark

Comment (breeswish) [reply ui/lib/apps/SearchLogs/components/SearchResult.tsx:2]: yeah, I did a global search & replace, so there will be some duplicate imports..

**Pull request merged**